Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -68,10 +68,19 @@ open class CarPlayNavigationViewController: UIViewController {
      */
     public private(set) var styleManager: StyleManager?
     
+    /**
+     Allows to control highlighting of the destination building on arrival. By default destination buildings will not be highlighted.
+     */
+    public var waypointStyle: WaypointStyle = .annotation
+    
     var mapTemplate: CPMapTemplate
     var carInterfaceController: CPInterfaceController
     
     private var isTraversingTunnel = false
+    private var approachingDestinationThreshold: CLLocationDistance = 250.0
+    private var passedApproachingDestinationThreshold: Bool = false
+    private var currentLeg: RouteLeg?
+    private var foundAllBuildings = false
```

Comment:
Let's use consistent style:
```suggestion
    private var foundAllBuildings: Bool = false
```

Reply:
Was also renamed to `buildingWasFound` to be more precise.